sources/packages/backend/apps/api/src/testHelpers/auth/institution-auth-helpers.ts

```diff
@@ -146,3 +146,6 @@ export const INSTITUTION_BC_PUBLIC_ERROR_MESSAGE =
 
 export const INSTITUTION_STUDENT_DATA_ACCESS_ERROR_MESSAGE =
   "The institution is not allowed access to the student data of the given student.";
+
+export const RESTRICTION_NOT_FOUND_ERROR_MESSAGE =
```

Comment:
The `INSTITUTION_BC_PUBLIC_ERROR_MESSAGE` and `RESTRICTION_NOT_FOUND_ERROR_MESSAGE` were added here because they would probably be used many times. I am not sue if we should start a practice of having all the error messages here.

Reply:
@andrepestana-aot @ann-aot @dheepak-aot @guru-aot  suggestions?

Reply:
I think we were adding the constants in `constants.ts`